monitoring/api/v3/alerts-client/snippets_test.py

```diff
@@ -102,15 +103,23 @@ def test_enable_alert_policies(capsys, pochan):
     def invoke_sample(val):
         snippets.enable_alert_policies(pochan.project_name, val)
 
+    # These sleep calls are for mitigating the following error:
+    # "409 Too many concurrent edits to the project configuration.
+    # Please try again."
+    # TODO(#3310): Having multiple projects will void these `sleep()` calls
+    time.sleep(2)
     invoke_sample(False)
+    time.sleep(2)
```

Comment:
Do we need to run the snippets twice? Why isn't once sufficient enough?

Reply:
Ask original author ;)

My understanding is that, the first call will disable it correctly, and the second call is for `assert "already disabled" in out`?

I'm just preserving the original test.

Reply:
If original test is flaky, why not replace it instead of trying to preserve the original?

Reply:
Removing one call won't help much, but I can remove some.